docs/conf.py

```diff
@@ -80,15 +80,19 @@
 # we need to define these package to URL mappings:
 intersphinx_mapping = {
     "arrow": ("https://arrow.apache.org/docs/", None),
+    "dagster": ("https://docs.dagster.io/", None),
```

Comment:
Noticed that we had not updated these links in a long time, and added new libraries that we depend on so that links to their docs from ours will work.